common/lib/capa/capa/responsetypes.py

```diff
@@ -1416,7 +1699,7 @@ def check_hint_condition(self, hxml_set, student_answers):
 
             if self.check_string([hinted_answer], given):
                 hints_to_show.append(name)
-        log.debug('hints_to_show = %s', hints_to_show)
+            log.debug('hints_to_show = %s', hints_to_show)
```

Comment:
Oh shoot, I think that's just an error on my part. Thomas had removed stuff, and I was trying as much as possible for this PR to just leave everything the way it was. Good catch!